e2e/e2e.js

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const execa = require('execa')
+const chalk = require('chalk')
+const { stdout } = require('stdout-stderr')
+const fs = require('fs')
+
+stdout.print = true
+
+test('Generate jwt-auth access token test', async () => {
+  const packagejson = JSON.parse(fs.readFileSync('package.json').toString())
+  const name = `${packagejson.name}`
+  console.log(chalk.blue(`> e2e tests for ${chalk.bold(name)}`))
+
+  console.log(chalk.dim(`    - Generate jwt-auth access token..`))
+  expect(() => { execa.sync('./bin/run', ['jwt-auth:access-token'], { stderr: 'inherit' }) }).not.toThrow()
```

Comment:
i'm pretty sure this will throw an exception.
run `npm run e2e` . Travis does not run this

Reply:
it requires ~/.wskprops

ieyu-osx:aio-cli-plugin-jwt-auth jieyu$ cat ~/.wskprops
APIHOST=https://adobeioruntime.net
NAMESPACE=53444_70361
AUTH=xxxxxx

Reply:
actually if you run it, it will complain about missing `jwt-auth` key.

Reply:
yes, need to fill out the config file: e2e/jwt_auth_config.json